src/RunTests/README.md

```diff
@@ -0,0 +1,30 @@
+# Microsoft.Build.RunTests
+
+The `Microsoft.Build.RunTests` MSBuild SDK adds support for running tests from MSBuild, similarly to how one would use `dotnet test`.
+
+## Usage in `Directory.Packages.Props`
+In your `Directory.Packages.props`:
+```xml
+<Project>
+  <ItemGroup>
+    <!-- <PackageVersion> elements here -->
+  </ItemGroup>
+  <ItemGroup>
+    <GlobalPackageReference Include="Microsoft.Build.RunTests" Version="1.0.0" />
+  </ItemGroup>
+</Project>
+```
+This example will include the `Microsoft.Build.RunTests` task for all NuGet-based projects in your repo.
+
+## Example
+To run tests
+```
+$env:MSBUILDENSURESTDOUTFORTASKPROCESSES=1
+msbuild /nodereuse:false /t:MSBuildRunTests
+```
+
+To build and run tests
+```
+$env:MSBUILDENSURESTDOUTFORTASKPROCESSES=1
+msbuild /nodereuse:false /t:Build;MSBuildRunTests
```

Comment:
Integration tests (at least ours) depend on nuget packages being built, and not all our projects generate packages during Build.

How is the order going to work when I also want to call Pack? Will the new task run before or after, or there is no guarantee? Ideally it would run after, but without taking dependency on Pack. Can this be achieved?

Reply:
The order would be determined by the user, for example this should work just fine: `msbuild /nodereuse:false /t:Build;Pack;MSBuildRunTests`